optaplanner-benchmark/src/test/resources/org/optaplanner/benchmark/config/testBenchmarkConfig.xml

```diff
@@ -44,7 +44,7 @@
       </localSearch>
     </solver>
     <problemBenchmarks>
-      <xStreamAnnotatedClass>org.optaplanner.core.impl.testdata.domain.TestdataSolution</xStreamAnnotatedClass>
+      <solutionFileIOClass>org.optaplanner.benchmark.config.PlannerBenchmarkConfigTest$TestdataSolutionFileIO</solutionFileIOClass>
```

Comment:
Can we make it a top-level class? This dollar syntax just makes it more confusing to me.
(Although it's nice to see that it works.)

Reply:
Sure we can, but the class has nothing to offer outside the test case. I already use several nested classes for the `SolverConfigTest` and would rather not make them standalone top-level classes, as they basically do nothing.